petrel/setup.py

```diff
@@ -59,8 +62,10 @@ def build_petrel():
     subprocess.check_call(['mvn', '-Dstorm_version=%s' % version_string, 'assembly:assembly'])
     os.chdir(old_cwd)
     shutil.copyfile(
-        '../jvmpetrel/target/storm-petrel-%s-SNAPSHOT.jar' % version_number,
+        '../jvmpetrel/target/storm-petrel-%s-SNAPSHOT-jar-with-dependencies.jar' % version_number,
```

Comment:
Not sure. It gets put into the jvmpetrel .jar, which is build outside the topology? So I'm assuming it's Petrel that wants it?